libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -4750,6 +4750,41 @@ void GCS_MAVLINK::send_set_position_target_global_int(uint8_t target_system, uin
             0,0);   // yaw, yaw_rate
 }
 
+void GCS_MAVLINK::send_position_target_global_int()
+{
+    uint16_t type_mask;
+    Location target;
+    Vector3f target_vel, target_accel;
+    if (!AP::vehicle()->get_target_info(type_mask, target, target_vel, target_accel)) {
```

Comment:
Why go via vehicle here?

Reply:
I don't really have to, but there was the scripting only function which `get_target_info()` is similar to just with more info.  Here, https://github.com/ArduPilot/ardupilot/blob/da3ff2c142b1e3a65d66533f0133da1249450245/libraries/AP_Vehicle/AP_Vehicle.h#L191